src/stores/gameBoard.ts

```
@@ -7,23 +7,23 @@ export const getBoards = (): Promise<DBGameBoard[]> => {
   return gameBoard.orderBy('title').toArray();
 };
 
-export const getActiveBoard = (): Promise<DBGameBoard | undefined> => {
-  return gameBoard.where('isActive').equals(1).first() || {};
+export const getActiveBoard = async (): Promise<{}> => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)